string taskId
string patientName
geometry_msgs/Pose patientPosition
